web/server/codechecker_server/auth/cc_ldap.py

```diff
@@ -410,8 +410,10 @@ def get_groups(ldap_config, username, credentials):
                               user_dn_postfix_preference)
 
         group_pattern = ldap_config.get('groupPattern', '')
```

Comment:
I think this could be changed too:
```suggestion
        group_pattern = ldap_config.get('groupPattern')
```

so the group_pattern will be None or some pattern string (which will not be an empty string)

Reply:
Will do. You prefer separate commit and squash in the end of review? Or amend my previous commit now?

Reply:
@jimis You can amend your commit.

Reply:
BTW, What is this function supposed to return? I see `return False` and `return []` in different parts. I'm inclined to change `return False` to `return None`, should I?